src/components/ListItem/ListItem.tsx

```diff
@@ -46,14 +69,21 @@ export const ListItem: React.FC<IListItemProps> = ({
     );
 
     return (
-        <li className={classnames(cssListItem)}
+        <li
+            className={classnames(cssListItem)}
+            tabIndex={tabIndex}
             {...props}
         >
             {
-            iconStart &&
+                iconStart &&
                 <iconStart.type {...iconStart.props} size="small" className={classnames('moonstone-listItem_iconStart')}/>
             }
 
+            {
+                image &&
```

Comment:
Do we prevent the possibility to have an icon and an image at the same time with something like `image && !iconStart` ?

Reply:
Same answer as above for `iconStart && !image` - could be confusing for the user but I'll add info to the props documentation.